### ISSUE TITLE
Tag colours need to change for accessibility

### ISSUE THREAD

**Issue by emollett** [nice-digital/nice-design-system#89]
We currently have an accessibility issues related to the colour used for the Active tag related to colour contrast. This should be resolved by change to the tags colour.

See attached for the colours used in IdAM for tags
Jira issue - IDAM-297

![localhost_3000_users](https://user-images.githubusercontent.com/18617190/72899613-468a3f80-3d1e-11ea-97d1-fdafd787e6cc.png)

**Comment by greypiglet** [comment]
This was done as part of the tag update in November

See here - https://github.com/nice-digital/nice-design-system/blame/6402c31d9ade93b685f13c7c73264f787d9e5619/packages/nds-core/scss/settings/_settings-colours-semantic.scss
